packages/cli/src/lib/quire/index.js

```diff
@@ -129,20 +117,44 @@ async function initStarter (starter, projectPath) {
  */
 async function install(version='latest', options={}) {
   console.debug(`[CLI:quire] installing quire-11ty@${version}`)
-  fs.ensureDirSync(INSTALL_PATH)
+  const absoluteInstallPath = path.join(__dirname, 'versions')
+  fs.ensureDirSync(absoluteInstallPath)
   /**
    * `Destination` is relative to `node_modules` of the working-directory
    * so we have included a relative path to parent directory in order to
    * install versions to a different local path.
    * @see https://github.com/scott-lin/install-npm-version
    */
-  const options = {
-    Destination: path.join('../', INSTALL_PATH, version),
+  const installOptions = {
+    Destination: path.join('../', version),
+    Debug: false,
     Overwrite: options.force || options.overwrite || false,
     Verbosity: options.debug ? 'Debug' : 'Silent',
+    WorkingDirectory: absoluteInstallPath
   }
-  await inv.Install(`${PACKAGE_NAME}@${version}`, options)
+  await inv.Install(`${PACKAGE_NAME}@${version}`, installOptions)
+
+  // delete empty `node_modules` directory that `install-npm-version` creates
+  const invNodeModulesDir = path.join(absoluteInstallPath, 'node_modules')
+  if (fs.existsSync(invNodeModulesDir)) fs.rmdir(invNodeModulesDir)
+
   symlinkLatest()
+
+  console.debug('[CLI:quire] installing dev dependencies')
+  /**
+   * Install dev dependencies manually, as they are necessary to run 11ty.
+   * They should be kept as dev dependencies so that they don't get bundled into
+   * the final compiled `_site` package when running `quire build`
```

Comment:
```suggestion
   * Manually install necessary dev dependencies to run 11ty;
   * these must be `devDependencies` so that they are not bundled into
   * the final `_site` package when running `quire build`
```